src/r_emit.c

```diff
@@ -274,7 +274,7 @@ static SEXP Ryaml_yoink(SEXP s_vec, int index)
       break;
     case INTSXP:
       if (factor) {
-        s_levels = GET_LEVELS(s_vec);
+        s_levels = getAttrib(s_vec, R_LevelsSymbol);
```

Comment:
Not using the `Rf_` prefix seems to be the style of this package, so I matched that.